clang/test/CodeGen/pr70496.c

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang --target=wasm32 -mmultivalue -Xclang -target-abi -Xclang experimental-mv %s -S -Xclang -verify
+
+float crealf() { return 0;} // expected-no-diagnostics
```

Comment:
Single `expected-no-diagnostics` is sufficient for the whole file

Reply:
ok, nice hit!

Reply:
Sorry, I actually meant that you can do like https://github.com/llvm/llvm-project/blob/main/clang/test/CXX/basic/basic.link/p6.cpp does, putting `// expected-no-diagnostics` after RUN lines. I didn't mean that you should reduce test coverage: I believe you had 4 cases, now you have just one.